docs/docs/faq.md

```diff
@@ -0,0 +1,103 @@
+- Wireless First
+  - ZMK is designed for the future, and we believe the future is wireless.  So power efficiency plays a critical role in every design decision, just like in Zephyr™.
+
+The ZMK contributors firmly believe that a keyboard firmware built on Zephyr™ will provide more long term benefits.
+
+### What license does ZMK use?
+
+ZMK uses the MIT [license](https://github.com/zmkfirmware/zmk/blob/master/LICENSE).
+
+### What hardware/platforms does ZMK support?
+
+ZMK has the potential to run on any platform supported by Zephyr™.  However, it’s impractical for the ZMK contributors to test all possible hardware.
+
+The Zephyr™ [documentation](https://docs.zephyrproject.org/latest/boards/index.html) describes which hardware is currently natively supported by the Zephyr™ platform.  *Similar documentation covering which keyboards have been integrated into ZMK is currently being planned.*
+
+### Does ZMK compile for AVR?
+
+Sorry, Zephyr™ only supports 32-bit and 64-bit platforms.
+
+### How do I get started?
+
+ZMK is still in its infancy, so there’s a learning curve involved.  But if you’d like to try it out, please check out the development [documentation](/docs) and the other FAQs.  Please keep in mind that the project team is still small, so our support capability is limited whilst we focus on development.  But we’ll try our best!  Interested developers are also very welcome to contribute!
+
+### What is a “board”?
+
+In ZMK, a *board* defines the *PCB* that *includes the MCU*.
+For keyboards, this is one of two options:
+- Complete keyboard PCBs that include the MCU (e.g. the Planck or Preonic).
+- Small MCU boards (e.g. the Proton-C or nice!nano) that expose pins and are designed to be combined with larger keyboard PCBs, or hand wired to switches to create the final keyboard.
+
+### What is a “shield”?
+
+In ZMK, a *shield* is a *PCB* or *hardwired set of components* that when combined with a MCU only board like the Proton-C or nice!nano, results in a complete usable keyboard. Examples would be keyboard PCBs like the Kyria or Corne.  The shield is usually the big PCB containing all the keys.
+
+### Why *boards* and *shields*?  Why not just “keyboard”?
+
+When a keyboard accepts a small “PCB MCU module” (e.g. *Arduino Pro Micro*) for its “brains”, then it's important to conceptually separate the hardware into a *board* PCB and a *shield* PCB.
+
+The *shield* is a brainless shell containing all the keys, RGB LEDs, encoders etc.  It maps all of these features to a standard pin footprint, such as the Pro Micro pinout.
+
+To bring this brainless *shield* to life, you attach any MCU *board* matching the footprint.  For instance, the *nice!nano* is *pin-compatible* with the *Arduino Pro Micro*, so you can substitute either *board* onto the *shield*.  But each *board* comes with its own features (MCU, flash, BLE, etc.) which must also be handled.
```

Comment:
Same thing here. Maybe link to "What is a board?" especially because this doesn't outline full completed keyboards.

Reply:
Added.